tests/eas/load_tracking.py

```diff
@@ -537,6 +540,371 @@ def test_load_avg_behaviour(self, experiment, tasks):
         """
         return self._test_behaviour(experiment, tasks, 'load_avg')
 
+class _CPUMigrationBase(LisaTest):
+    """Base class for shared functionality of tasks migration tests"""
+    test_conf = {
+        'tools'    : [ 'rt-app' ],
+        'ftrace' : {
+            'events' : [
+                'sched_switch',
+                'sched_load_cfs_rq',
+            ],
+        },
+        'modules': ['cpufreq', 'cgroups'],
+    }
+
+    # Allowed error margin
+    allowed_util_margin = UTIL_SCALE * 0.02
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        super(_CPUMigrationBase, cls).runExperiments(*args, **kwargs)
+
+    @classmethod
+    def get_wload(cls, test_env, tasks, workload_name):
+        """Create a specification for a rt-app workload from a list of tasks
+        :param test_env: Environment of the test
+        :type test_env: TestEnv
+
+        :param tasks: lists of tasks
+        :type tasks:[RTATask]
+
+        :param workload_name: name of the workload
+        :type workload_name: str
+
+        :return the workload specification
```

Comment:
AFAICT the syntax for the "return" statement should be:
`:returns:  the workload specification`
(https://stackoverflow.com/a/24385103)

Not sure if this would create an actual error when generating the doc. I noticed that our sphinx setup doesn't build the doc for the tests, I need to fix that.

Reply:
Done